src/plugins/console/public/application/containers/editor/monaco/monaco_editor_actions_provider.ts

```diff
@@ -117,7 +117,7 @@ export class MonacoEditorActionsProvider {
       const offset = this.editor.getTopForLineNumber(lineNumber) - this.editor.getScrollTop();
       this.setEditorActionsCss({
         visibility: 'visible',
-        top: offset,
+        top: offset - 2, //  Move position up by 2 px
```

Comment:
nit: can we add a little description of why this is needed?

Reply:
Sure, added in [2e5e850](https://github.com/elastic/kibana/pull/190843/commits/2e5e850048c18178d78cb24c10e055d83e438a24)!